interface/Gruntfile.js

```diff
@@ -96,6 +98,37 @@ module.exports = function ( grunt ) {
     ],
 
     /**
+     * Browserify the ttest library since it's written just for node
+     * Note: this must be run before the copy step (see `build` below).
+     */
+    browserify: {
+      dist: {
+        src: 'node_modules/ttest/hypothesis.js',
+        dest: 'node_modules/ttest/ttest.js',
```

Comment:
(Forgive me if I sound too sensitive.) Do you think we should choose a more specific name for the destination file? (Maybe something like `ttest_greenelab.js`). I am a little worried that the author of this package might create his own *ttest.js* file in the future for whatever reason.